### DISCUSSION TITLE
I want konw how to use the NPU of the AMD 370 for acceleration

### DISCUSSION THREAD

**Discussion by whycloud2023** [oobabooga/text-generation-webui#6379] [Q&A]
I noticed that AMD's new CPU 370 has enhanced the NPU, and it is officially claimed to provide 80 TOPS of computing power. How should I configure it to utilize this computational power?

**Comment by Tedy50** [comment]
I doubt if that NPU has any use because performance is more limited by memory bandwidth, even average CPU is not fully utilized. NPU can only improve power consumption
